documentation/command/baseline.md

```diff
@@ -13,7 +13,32 @@ Baseline is for introducing Flyway to [existing databases](/documentation/learnm
 at a specific version. This will cause [Migrate](/documentation/command/migrate) to ignore all migrations
 up to and including the baseline version. Newer migrations will then be applied as usual.
 
+## Usage
 See [configuration](/documentation/configuration/parameters/#baseline) for baseline specific configuration parameters.
+<div class="tabbable">
```

Comment:
Instead of copy-pasting this 7 times it might be better to make it an include? It seems like the only difference is the command which we can then pass through to the include

Reply:
See latest commit